cloudsmith_cli/cli/commands/push.py

```diff
@@ -54,6 +58,82 @@ def validate_upload_file(ctx, opts, owner, repo, filepath, skip_errors):
     return md5_checksum
 
 
+def init_multipart_upload(opts, owner, repo, filepath):
```

Comment:
We generally keep the `commands` files clean, and utils etc get moved to the `files.py` module - feels like that might be a better home for this?

Reply:
Code blocks moved